### ISSUE TITLE
Atom Beta gets stuck in auto-update loop.

### ISSUE THREAD

**Issue by dengeist** [atom/atom#12220]
### Description

Atom 1.9.0-beta1 gets stuck in a restart-to-install loop. I know 1.9b1 is not the latest but considering the nature of this bug, I suppose experiencing it on the latest build would be impossible.

### Steps to Reproduce

1. Open Atom menu
2. Click 'Restart to install update'
3. Atom restarts
4. The Atom menu shows that the same update is downloading again.

**Expected behavior:** Atom should install the update it downloads and see that there are no newer versions.

**Actual behavior:** In addition to the steps above, this behavior repeats even after downloading the beta disk image from the atom website.

### Versions

```
OS X: 10.11.6 (15G31)

$ atom-beta -v
Atom    : 1.9.0-beta1
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0

$ apm-beta -v
apm : 1.10.0
npm:  2.13.3
node: 0.10.40
python: 2.7.11
git: 2.8.1
```

**Comment by idleberg** [comment]
I experienced this on OS X 10.7, but since it's a work computer where I don't have administrative rights, it's the behaviour I'd expect. However, it would be nice if Atom would request administrative rights _before_ downloading anything – or at least store previously downloaded updates. Now, every time I check whether there's an update, Atom starts downloading it, unaware of the fact that I can't install it.

**Comment by 50Wliu** [comment]
I believe this might be because Atom 1.9.0-beta1 is not currently published (perhaps it might have been, then got reverted to Draft state).

@idleberg you are experiencing a different bug: #2860.

**Comment by thomasjo** [comment]
As pointed out by @50Wliu, the original report is caused by some kind of hiccup/mistake/test that ended up publishing Atom 1.9.0-beta1, which has since been fixed/reverted. The problem can be resolved by reinstalling 1.9.0-beta0.
